### ISSUE TITLE
Make all value accessors be properties

### ISSUE THREAD

**Issue by cprice404** [momentohq/client-sdk-php#48]

**Comment by cprice404** [comment]
@poppoerika this may be a no-op for PHP.  I just blindly created tickets for .NET and PHP at the same time.

**Comment by poppoerika** [comment]
@cprice404 @pgautier404 
As Chris suspected, it doesn't seem that PHP has the idea of property getter/setter.
There are something called "magic methods" but those are called for ALL inaccessible properties in a class which is not what we're trying to accomplish here.
Another thing I saw was to make `private` or `protected` properties `public` to make it like getter/setter, which I also think it's not what we want to accomplish here.
So I will stick to what we have and rename our `value`  methods to `valueString` to align with the .NET SDK.

If my understanding is correct, we can close this ticket.

**Comment by cprice404** [comment]
I did a poor job of phrasing this ticket.  I was just creating dupe tickets for both .NET and PHP at the same time.  In .NET, it was "we should use properties".  But the generic phrasing would be "we should pick the right mechanism for the language at hand, and make sure that we have applied it consistently."  I defer to Pete on what that mechanism should be for PHP, and the acceptance criteria for this ticket is just "make sure we are doing that consistently."
